distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json.enso

```diff
@@ -168,18 +168,26 @@ type JS_Object
     field_names self =
         Vector.from_polyglot_array (get_property_names self.js_object)
 
+    ## GROUP Conversions
+       Maps ths object to a Vector of Pairs.
+    map : (Text->Any->Any) -> Vector
+    map self action=[_,_] =
```

Comment:
Removed but it ends up as `a->b->[a,b]`.

Been around for ages.

Reply:
I'm not sure why we should remove it. It is supported syntax.

@JaroslavTulach This is a special case of https://github.com/enso-org/enso/blob/develop/docs/syntax/function-arguments.md#underscore-arguments

i.e. for `a-> b-> foo 1 a 2 b 3` can be replaced by `foo 1 _ 2 _ 3`.

This is just a special case where instead of a regular function call, we have it used in a vector literal.

Reply:
I think this was inspired by https://docs.scala-lang.org/scala3/book/fun-anonymous-functions.html